ajax/libs/griddle-react/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
@pvnr0082t this could be a practice for you, write a test to make sure the order of package.json, don't need to reach a perfect status at one step, but start with dealing the most important info first.